<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenuAsignatura</class>
 <widget class="QWidget" name="MenuAsignatura">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>514</width>
    <height>313</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Menu Asignaturas</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../icons/icono-titulacion.png</normaloff>../icons/icono-titulacion.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Regresar</string>
     </property>
     <property name="autoDefault">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="pushButton_11">
     <property name="text">
      <string>Guardar</string>
     </property>
     <property name="autoDefault">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Crear</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="1">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Nombre:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Programa:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLineEdit" name="lineEdit"/>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Creditos:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLineEdit" name="lineEdit_3"/>
       </item>
       <item row="1" column="2">
        <widget class="QLineEdit" name="lineEdit_2"/>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>Crear</string>
         </property>
         <property name="autoDefault">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Mostrar</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QTextBrowser" name="textBrowser"/>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="pushButton_3">
         <property name="text">
          <string>Mostrar</string>
         </property>
         <property name="autoDefault">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_9">
      <attribute name="title">
       <string>Buscar Programa</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_10">
       <item row="0" column="0" colspan="3">
        <widget class="QTextBrowser" name="textBrowser_8"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Programa:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEdit_12"/>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="pushButton_10">
         <property name="text">
          <string>Buscar</string>
         </property>
         <property name="autoDefault">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Buscar</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="1" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Código</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEdit_4"/>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="pushButton_4">
         <property name="text">
          <string>Buscar</string>
         </property>
         <property name="autoDefault">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="3">
        <widget class="QTextBrowser" name="textBrowser_2"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Modificar</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_8">
       <item row="0" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Código:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEdit_8"/>
       </item>
       <item row="0" column="2" rowspan="5">
        <widget class="QTextBrowser" name="textBrowser_6"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Nombre:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEdit_9"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Programa:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lineEdit_10"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Creditos:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="lineEdit_11"/>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QPushButton" name="pushButton_8">
         <property name="text">
          <string>Modificar</string>
         </property>
         <property name="autoDefault">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>Eliminar</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0" colspan="3">
        <widget class="QTextBrowser" name="textBrowser_3"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Código</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEdit_5"/>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="pushButton_5">
         <property name="text">
          <string>Eliminar</string>
         </property>
         <property name="autoDefault">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_6">
      <attribute name="title">
       <string>Desactivar</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0" colspan="3">
        <widget class="QTextBrowser" name="textBrowser_4"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Código:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEdit_6"/>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="pushButton_6">
         <property name="text">
          <string>Desactivar</string>
         </property>
         <property name="autoDefault">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_7">
      <attribute name="title">
       <string>Activar</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_7">
       <item row="0" column="0" colspan="3">
        <widget class="QTextBrowser" name="textBrowser_5"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Código:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEdit_7"/>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="pushButton_7">
         <property name="text">
          <string>Activar</string>
         </property>
         <property name="autoDefault">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_8">
      <attribute name="title">
       <string>Ordenar</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_9">
       <item row="0" column="0">
        <widget class="QPushButton" name="pushButton_9">
         <property name="text">
          <string>Ordenar</string>
         </property>
         <property name="autoDefault">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QTextBrowser" name="textBrowser_7"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>lineEdit</tabstop>
  <tabstop>lineEdit_2</tabstop>
  <tabstop>lineEdit_3</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>textBrowser</tabstop>
  <tabstop>pushButton_3</tabstop>
  <tabstop>textBrowser_2</tabstop>
  <tabstop>lineEdit_4</tabstop>
  <tabstop>pushButton_4</tabstop>
  <tabstop>lineEdit_8</tabstop>
  <tabstop>lineEdit_9</tabstop>
  <tabstop>lineEdit_10</tabstop>
  <tabstop>lineEdit_11</tabstop>
  <tabstop>pushButton_8</tabstop>
  <tabstop>textBrowser_6</tabstop>
  <tabstop>textBrowser_3</tabstop>
  <tabstop>lineEdit_5</tabstop>
  <tabstop>pushButton_5</tabstop>
  <tabstop>textBrowser_4</tabstop>
  <tabstop>lineEdit_6</tabstop>
  <tabstop>pushButton_6</tabstop>
  <tabstop>textBrowser_5</tabstop>
  <tabstop>lineEdit_7</tabstop>
  <tabstop>pushButton_7</tabstop>
  <tabstop>pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
